thirty-five-mm/app/assets/javascripts/application.js

```diff
@@ -13,3 +13,129 @@
 //= require jquery
 //= require jquery_ujs
 //= require_tree .
+
+$(document).ready(function() {
```

Comment:
Better to put this in another file alongside application.js rather than add to application.js - it's intended to be a manifest file that just lists the other files to include.